requirements.txt

```diff
@@ -16,12 +16,13 @@ pandas==0.23.4
 patsy==0.5.0
 prov==1.5.0
 psutil==5.4.6
-pygraphviz==1.3.1
+# pygraphviz==1.3.1
 python-dateutil==2.7.3
```

Comment:
?

Reply:
this is a cumbersome to install on Mac- I couldnt make it right on @HechengJin0 laptop.

we should move it to another pull request, but removing graphviz as a dependency would be nice. so we just move to another workflow plotting system.

Reply:
Got it, thanks!